twine/commands/check.py

```diff
@@ -106,7 +109,26 @@ def check(
     output_stream: IO[str] = sys.stdout,
     strict: bool = False,
 ) -> bool:
-    """Print output of checking."""
+    """Check the file(s) given by user.
+    
+    If there are no files to check, False is returned with a output.
+    
+    If the check is failed, error text is given and failure = True is returned.
+    
+    If strict is set to True by user, the check will fail when there are warnings.
+    Otherwise, the check will pass with warnings.
+    
+    If the check is passed, message for passing the check is given
+    and failure = False is returned.
+
+    Args:
+        dists (List[str]): the distribution files we are going to check
+        output_stream (IO[str], optional): Output stream of the check. Defaults to sys.stdout.
+        strict (bool, optional): Strict mode for the check. Defaults to False.
+
+    Returns:
+        bool: Determine whether the check has passed.
```

Comment:
I believe we are not using this format for docstrings but instead the Sphinx/reStructuredText style. Please correct it everywhere

Reply:
Fixed!